db/migrate/20211022131607_drop_cockpit.rb

```diff
@@ -0,0 +1,31 @@
+class DropCockpit < ActiveRecord::Migration[6.0]
+  class AssignedServerRole < ActiveRecord::Base; end
+
+  class ServerRole < ActiveRecord::Base; end
+
+  class SettingsChange < ActiveRecord::Base; end
+
+  class MiqUserRole < ActiveRecord::Base; end
+
+  class MiqRolesFeature < ApplicationRecord; end
+
+  class MiqProductFeature < ApplicationRecord
+    # habtm :miq_user_roles :join_table => MiqRoleFeature
```

Comment:
Love this comment :+1:

```suggestion
    # habtm :miq_user_roles :join_table => MiqRolesFeature
```